src/library/AssetMethods.ts

```diff
@@ -122,16 +122,24 @@ export class AssetMethods {
 
   /**
   * Fetch user transactions statistics
-  * @param {string} interval Interval of the input
   * @param {string} startDate Start date of the input
   * @param {string} endDate End date of the input
   * @public
   * @methods
   */
-  getUserStats = async (interval: string, startDate: string, endDate: string) => {
-    // console.debug("sdk getUserStats", interval, startDate, endDate);
-    const stats = await getUserTxStats(this.options.provider, this.options.account, interval, startDate, endDate)
-    return stats;
+  getUserStats = async (startTimestamp: number, endTimestamp: number) => {
```

Comment:
because this is not related to any asset, it shouldnt be on this file, this file is only assets methods related